bindings/python/triplestore/interface.py

```diff
@@ -0,0 +1,33 @@
+"""Triplestore interface."""
+from typing import TYPE_CHECKING, Protocol
+
+if TYPE_CHECKING:  # pragma: no cover
+    from collections.abc import Sequence
+    from typing import Generator
+
+    from dlite.triplestore import Triple
+
+
+class ITriplestore(Protocol):
```

Comment:
Consider adding an example using this class

Reply:
I have no clue of how to use it (except for inspection). ITriplestore is just a protocol class that formally define the interface that should be implemented by the. I added som additional documentation explaining this.